src/sgame/sg_cmds.cpp

```diff
@@ -3497,6 +3497,116 @@ void Cmd_Reload_f( gentity_t *ent )
 	}
 }
 
+/*
+=================
+Cmd_TeamStatus_f
+=================
+*/
+void Cmd_TeamStatus_f(gentity_t * ent)
+{
+	char multiomrc[12];
+	int builders = 0;
+	int arm = 0, mediboost = 0, miner = 0;
+	int omrccount = 0, omrchealth = 0;
+	bool omrcbuild = false;
+	gentity_t *tmp;
+	int i;
+
+	if (!g_teamStatus.Get()) 
+	{
+		trap_SendServerCommand(ent - g_entities, "print \"teamstatus is disabled.\n\"");
+		return;
+	}
+
+	if (ent->client->pers.namelog->muted) 
+	{
+		trap_SendServerCommand(ent - g_entities, "print \"You are muted and cannot use message commands.\n\"");
+		return;
+	}
+
+	if (ent->client->pers.lastTeamStatus &&
+		(level.time - ent->client->pers.lastTeamStatus) <
+	    g_teamStatus.Get() * 1000) 
+	{
+		trap_SendServerCommand(ent - g_entities, va("print \"You may only check your team's status once every %i seconds.\"", g_teamStatus.Get()));
+		return;
+	}
+
+	ent->client->pers.lastTeamStatus = level.time;
+
+	tmp = &g_entities[0];
+	for (i = 0; i < level.num_entities; i++, tmp++) 
+	{
+		if (i < MAX_CLIENTS) {
+			if (tmp->client &&
```

Comment:
This is just me, maybe, but, to have the `&& on start of line makes later git changes less likely to conflict, and quick'n dirty tweaking like commenting a line are also less changes.
I'll let other talk on this one though.